ee_tests/src/che_integration.spec.ts

```diff
@@ -42,7 +43,7 @@ describe('e2e_che_integration', () => {
 
   afterAll(async () => {
     logger.info('--- After all ---');
-    if (browser.params.reset.environment === 'true') {
+    if (specContext.resetEnvironment()) {
```

Comment:
`resetEnvironment` feels like it performs the reset... I would suggest something like `shouldResetEnvironment` or `getResetEnvironment`